spec/numberformat.html

```diff
@@ -30,7 +30,11 @@ <h1>SetNumberFormatDigitOptions ( _intlObj_, _options_, _mnfdDefault_, _mxfdDefa
           1. Set _intlObj_.[[MaximumSignificantDigits]] to _mxsd_.
         1. Else if _mnfd_ is not *undefined* or _mxfd_ is not *undefined*, then
           1. Set _intlObj_.[[RoundingType]] to ~fractionDigits~.
-          1. Let _mnfd_ be ? DefaultNumberOption(_mnfd_, 0, 20, _mnfdDefault_).
+          1. If _mxfd_ is not *undefined*, then
+            1. Let _mnfdActualDefault_ be min( _mxfd_, _mnfdDefault_ ).
```

Comment:
This doesn't work, because `mxfd` is any [ECMAScript language value](https://tc39.es/ecma262/#sec-ecmascript-language-types), but [min](https://tc39.es/ecma262/#eqn-min) requires its inputs to be numbers (excluding NaN, but including infinities).

Reply:
@anba realized this once I checked an implementation again. How does this look?

Reply:
Checking `Type(mxfd) is Number` goes against the normal conventions within ECMA-402/262, the `ToNumber` conversion should happen up front. Furthermore if `mxfd` is negative, `mnfdActualDefault` will be negative, too. And then `mnfd` will also be negative. And that will surely lead to issues elsewhere.